lib/internal/Magento/Framework/App/Config/ScopeCodeResolver.php

```diff
@@ -34,10 +34,10 @@ public function __construct(ScopeResolverPool $scopeResolverPool)
      * Resolve scope code
      *
      * @param string $scopeType
-     * @param string $scopeCode
+     * @param null|string $scopeCode
      * @return string
      */
-    public function resolve($scopeType, $scopeCode)
+    public function resolve($scopeType, $scopeCode = null)
```

Comment:
We are not allowed to modify public methods signature according to our [Backward compatible development guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)